BRLS/src/library/Main.java

```diff
@@ -7,265 +7,250 @@
 import library.fixitem.fIX_iTeM_cONTROL;
 import library.payfine.PayFineUI;
 import library.payfine.pAY_fINE_cONTROL;
-import library.returnBook.ReturnBookUI;
-import library.returnBook.rETURN_bOOK_cONTROL;
-
+import library.returnBook.ReturnItemUI;
+import library.returnBook.ReturnItemControl;
 import java.text.SimpleDateFormat;
 import java.util.Scanner;
 
-
 public class Main {
-	
-	private static Scanner SCANNER;
-	private static Library LIBRARY;
-	private static Calendar CALENDAR;
-	private static SimpleDateFormat SIMPLEDATEFORMAT;
-	
-	private static String MENU = """
-		Library Main Menu
-		
-			AP  : add patron
-			LP : list patrons
-		
-			AI  : add item
-			LI : list items
-			FI : fix item
-		
-			B  : borrow an item
-			R  : return an item
-			L  : list loans
-		
-			P  : pay fine
-		
-			T  : increment date
-			Q  : quit
-		
-		Choice : 
-		""";		
+    
+    private static Scanner scanner;
+    private static Library library;
+    private static Calendar calendar;
+    private static SimpleDateFormat simpleDateFormat;
+    
+    private static String menu = """
+        Library Main Menu
+        
+            AP  : add patron
+            LP : list patrons
+        
+            AI  : add item
+            LI : list items
+            FI : fix item
+        
+            B  : borrow an item
+            R  : return an item
+            L  : list loans
+        
+            P  : pay fine
+        
+            T  : increment date
+            Q  : quit
+        
+        Choice : 
+        """;        
+
+    public static void main(String[] args) {        
+        try {            
+            scanner = new Scanner(System.in);
+            library = Library.getInstance();
+            calendar = Calendar.getInstance();
+            simpleDateFormat = new SimpleDateFormat("dd/MM/yyyy");
+    
+            for (Patron patron : library.listPatrons()) {
+                displayOutput(patron);
+            }
+            displayOutput(" ");
+            for (Item item : library.listItems()) {
+                displayOutput(item);
+            }
+                        
+            boolean finished = false;
+            
+            while (!finished) {
+                
+                displayOutput("\n" + simpleDateFormat.format(calendar.getDate()));

```

Comment:
PR3 D12

The argument list of the displayOutput contains a reference to simpleDateFormat.format(), which also contains a reference to calendar.getDate(). This is in violation of Rule 10 of the ITC205 code style guidelines, and should be split across three lines.

Reply:
I think I have now addressed this.

Reply:
Unfortunately not, because displayOutput() still contains a reference to simpleDateFormat.format(). One more line split is required.

Reply:
fixed it. Please review.

Reply:
I... don't see a commit. Have you pushed to main?